builtin/providers/google/resource_dns_managed_zone.go

```diff
@@ -32,7 +34,6 @@ func resourceDnsManagedZone() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
-				Default:  "Managed by Terraform",
```

Comment:
Eek! 
`resource google_dns_managed_zone: description: Default must be nil if computed`
I'll leave the default as-is. Users will have to manually provide an empty description on imported managed zones.